poc_iot_verifier/src/loader.rs

```diff
@@ -184,7 +184,7 @@ impl Loader {
                     id_hash,
                     packet_data,
                     buf_as_vec,
-                    &datetime_from_epoch(event.timestamp),
+                    &event.timestamp.to_timestamp_nanos()?,
```

Comment:
why are we doing witness ingest report to nanos but beacon ingest to millis?

Reply:
this branch is stale, fixed in latest